src/index.html

```diff
@@ -8,9 +8,51 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <div class="page__search-bar">
+      <form
+        class="search-bar search-bar--large"
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <fieldset class="search-bar__rectangle">
+          <button type="submit" class="search-bar__search-button"></button>
```

Comment:
It shouldn't be a button. It's just an icon. Also, better make it the background image of input
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/4d78c6f3-8476-4171-aebe-38f15c0e8488)